### ISSUE TITLE
can't connect my c-lightining node

### ISSUE THREAD

**Issue by renepickhardt** [shocknet/api#449]
wanted to test the service but it only works with your hosted nodes and lnd

maybe a plugin for c-lightning would be useful?

**Comment by capitalistdog** [comment]
Yes, it is currently very tightly reliant on LND in its current state primarily due to the authentication. We are however working to de-couple that authentication mechanism which would allow for adapters to C-LN behind it.

If anyone is interested in taking that on I'm happy to kick in on a bounty.
